cinema/management/wait_for_db.py

```diff
@@ -0,0 +1,20 @@
+import time
+
+from django.core.management import BaseCommand
+from django.db import connections, OperationalError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database...")
```

Comment:
It's a good practice to add a docstring to your classes and methods to explain what they do. For example, you could add a docstring to the `Command` class and the `handle` method to explain their purposes in the context of your application.